app/helpers/application_helper.rb

```diff
@@ -28,4 +28,8 @@ def default_meta_tags
       reverse: true
     }
   end
+
+  def timeago(time)
+    content_tag(:span, time.in_time_zone('Kyiv').strftime("%m/%d/%Y at %R"))
```

Comment:
make Kyiv as default time zone in application controller

and use helper
= I18n.l time, format: :long

(read about formats)

remove this method